src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -34,50 +34,16 @@
-                                            <controls:TextBlockControl x:Uid="ColorPickerFirst_Description" 
-                                                                       FontSize="{StaticResource SecondaryTextFontSize}"
-                                                                       EnabledForeground="{ThemeResource TextFillColorSecondaryBrush}"
-                                                                       DisabledForeground="{ThemeResource TextFillColorDisabledBrush}"/>
-                                        </StackPanel>
-                                </RadioButton.Content>
-                            </RadioButton>
-
-                            <RadioButton x:Uid="EditorFirst_Accessible" IsChecked="{Binding ActivationOpensEditor, Mode=TwoWay}" GroupName="ColorPickerActivationAction">
-                                <RadioButton.Content>
-                                    <StackPanel>
-                                        <TextBlock x:Uid="EditorFirst"/>
-                                            <controls:TextBlockControl x:Uid="EditorFirst_Description" 
-                                                                       FontSize="{StaticResource SecondaryTextFontSize}"
-                                                                       EnabledForeground="{ThemeResource TextFillColorSecondaryBrush}"
-                                                                       DisabledForeground="{ThemeResource TextFillColorDisabledBrush}"/>
-                                    </StackPanel>
-                                </RadioButton.Content>
-                            </RadioButton>
-
-                            <RadioButton x:Uid="ColorPickerOnly_Accessible" IsChecked="{Binding ActivationOpensColorPickerOnly, Mode=TwoWay}" GroupName="ColorPickerActivationAction">
-                                <RadioButton.Content>
-                                    <StackPanel>
-                                            <TextBlock x:Uid="ColorPickerOnly"/>
-                                            <controls:TextBlockControl x:Uid="ColorPickerOnly_Description" 
-                                                                       FontSize="{StaticResource SecondaryTextFontSize}"
-                                                                       EnabledForeground="{ThemeResource TextFillColorSecondaryBrush}"
-                                                                       DisabledForeground="{ThemeResource TextFillColorDisabledBrush}"/>
-                                        </StackPanel>
-                                </RadioButton.Content>
-                            </RadioButton>
-                        </StackPanel>
-                    </controls:SettingExpander.Content>
-                </controls:SettingExpander>
+                            <controls:Setting x:Uid="ColorPicker_ActivationAction" Icon="&#xEC4E;" >
```

Comment:
```suggestion
                            <controls:Setting x:Uid="ColorPicker_ActivationAction" Icon="&#xE8A7;" >
```
![Image](https://docs.microsoft.com/de-de/windows/apps/design/style/images/glyphs/segoe-mdl-2/e8a7.png)

Reply:
We already use this icon for e.g. the Launch FZ Editor button (as W11 Settings does for all external links or apps), so hesitant to use it for this.